### PR TITLE
[Excel] (Custom Functions) Clarify runtimes in UI-less debugging article

### PR DESCRIPTION
**Pull request by alison-mk**
Updating article based on customer issue #2597.

### CODE REVIEW

File: docs/excel/custom-functions-debugging.md [hunk 1]
```diff
@@ -27,7 +29,7 @@ On Mac:
 
 ## Requirements
 
-Before starting to debug, you should use the [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office) to create a custom functions project. For guidance about how to create a custom functions project, see the [custom functions tutorial](../tutorials/excel-tutorial-create-custom-functions.md).
+This debugging process only works for UI-less custom functions, which are custom functions that use the `OfficeRuntime` JavaScript runtime. This debugging process doesn't work with custom functions that use the `SharedRuntime` JavaScript runtime, or with custom functions that use the `CustomFunctionsRuntime` JavaScript runtime.
```
Comment (mscharlock) [inline docs/excel/custom-functions-debugging.md:32]:
```suggestion
This debugging process only works for UI-less custom functions, which are custom functions that use the `OfficeRuntime` JavaScript runtime. This debugging process doesn't work with custom functions that use the `SharedRuntime` JavaScript runtime. 
```

File: docs/excel/custom-functions-debugging.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
-ms.date: 07/10/2020
+ms.date: 04/08/2021
 description: Learn how to debug your Excel custom functions that don't use a task pane.
 title: UI-less custom functions debugging
 localization_priority: Normal
 ---
 # UI-less custom functions debugging
 
-Debugging for custom functions that don't use a task pane or other user interface elements (UI-less custom functions) can be accomplished by multiple means, depending on what platform you're using.
+Debugging for custom functions that don't use a task pane or other user interface elements (UI-less custom functions) can be accomplished by multiple means, depending on what platform you're using.
```
Comment (mscharlock) [inline docs/excel/custom-functions-debugging.md:9]:
```suggestion
This article discusses debugging *only* for custom functions that don't use a task pane or other user interface elements (UI-less custom functions). 
```

**Review by mscharlock — CHANGES_REQUESTED** [review]
Does this make sense? Happy to chat about it as well.

**Comment by alison-mk** [comment]
Thanks for the review, @mscharlock! Ready for you to take another look when you have a chance.

**Review by mscharlock — APPROVED** [review]

**Pull request merged**